packages/fxa-auth-server/lib/senders/emails/layouts/subscription/index.mjml

```diff
@@ -10,6 +10,7 @@
       <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
     </mj-raw>
     <mj-title><%- subject %></mj-title>
+    <%- include('/partials/images.mjml') %>
```

Comment:
Commenting here to cross-link:

We should document how to add alt text to images and why we're doing it this way. We can do this in FXA-4124 / #10758 or FXA-4062 / #10617